bundles/org.openhab.binding.resol/README.md

```diff
@@ -0,0 +1,56 @@
+# Resol Binding
+
+Resol Binding connects to Solar and System Controllers of RESOL - Elektronische Regelungen GmbH, also including branded versions from Viessmann, SOLEX, COSMO, SOLTEX, DeDietrich and many more.
+
+This binding is based on and includes the [Resol-VBUS-Java library](https://github.com/danielwippermann/resol-vbus-java), developed by Daniel Wippermann.
+
+## Supported Things
+
+VBusLAN-Bridge, DataLogger DL2 and DL3 as a live data interface between LAN and Resol VBus. On the DL3 currently there is only the first VBUS channel supported and the sensors directly connected tot he DL3 are not accessible via this binding.
+
+On top of the bridge devices, which enables access to the VBUS many, if not all, Resol Controllers and Modules like WMZ heat meters, HKM Heating circuit extensions etc. are supported including branded versions from different suppliers.
+
+## Discovery
+
+Discovery is tested for VBus-LAN adapters DL2, DL3 and KM2 devices, it should also work for other devices providing a live data port.
+
+## Binding Configuration
+
+The Resol binding doesn't need any form of configuration in files.
+
+## Thing Configuration
```

Comment:
Yes I need to add a thing file configuration example. Although it will be hard, as it is quite dependent on the connected devices and with the vbus-library the available channels are dynamically configured, so I'll put it for the device I own.

Reply:
Making an example for a device you own is a good approach. Additionally there should be a Chapter, where all Things are described, including their configuration parameters. Like here https://www.openhab.org/addons/bindings/telegram/#thing-configuration